docs/en/operations/settings/settings.md

```diff
@@ -242,13 +242,14 @@ Default value: 1.
 
 Sets default strictness for [JOIN clauses](../../query_language/select.md#select-join).
 
-**Possible values**
+Possible values:
 
 - `ALL` — If the right table has several matching rows, the data is multiplied by the number of these rows. This is the normal `JOIN` behavior from standard SQL.
 - `ANY` — If the right table has several matching rows, only the first one found is joined. If the right table has only one matching row, the results of `ANY` and `ALL` are the same.
+- `ASOF` — For joining sequences with an uncertain match.
 - `Empty string` — If `ALL` or `ANY` is not specified in the query, ClickHouse throws an exception.
```

Comment:
ASOF made as separate strictness. User may set one of {ALL, ANY, ASOF}. It would be ALL if not set. There's also join_default_strictness, but I don't think we should tell user about it cause it's not OK to change it's value (it's backward compatibility only setting).